File: source/kCura.Relativity.Export/Exporters/TempTextFileDeletor.vb
Namespace kCura.Relativity.Export.Exports
	Public Class TempTextFileDeletor
		Private _filepaths As System.Collections.Generic.IEnumerable(Of String)
		Public Sub New(ByVal filePaths As System.Collections.Generic.IEnumerable(Of String))
			_filepaths = filePaths
			If _filepaths Is Nothing Then _filepaths = New String() {}
		End Sub
		Public Sub DeleteFiles()
			For Each path As String In _filepaths
				If Not String.IsNullOrEmpty(path) Then kCura.Utility.File.Instance.Delete(path)
			Next
		End Sub
	End Class
End Namespace

File: source/kCura.Relativity.Export.ConvertedToC#/Exporters/TempTextFileDeletor.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Exports
{
	public class TempTextFileDeletor
	{
		private System.Collections.Generic.IEnumerable<string> _filepaths;
		public TempTextFileDeletor(System.Collections.Generic.IEnumerable<string> filePaths)
		{
			_filepaths = filePaths;
			if (_filepaths == null)
				_filepaths = new List<string>();
		}
		public void DeleteFiles()
		{
			foreach (string path in _filepaths) {
				if (!string.IsNullOrEmpty(path))
					kCura.Utility.File.Instance.Delete(path);
			}
		}
	}
}
